### PR TITLE
7.x 1.x

### PR DESCRIPTION
**Pull request by fabianx-ai**

### CODE REVIEW

File: file_dropzone.module [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+/**
+ * Process callback for the file managed field element.
+ */
+function file_dropzone_file_field_widget_process($element, &$form_state, $form) {
+  $field = field_widget_field($element, $form_state);
+
+  // Adjust the Ajax settings so that on upload and remove of any individual
+  // file, the entire group of file fields is updated together.
+  // Also add our file_dropzone menu callback.
+  // @todo Make cardinality 1 work.
```
Comment (damiankloip) [inline file_dropzone.module:411]: Wouldn't this currently cause issues for anything other than CARDINALITY_UNLIMITED as more files could be uploaded than is allowed? So could just check for that?

File: file_dropzone.info [hunk 2]
```diff
@@ -0,0 +1,7 @@
+name = File Dropzone
+description = Provides a dropzone for every multiple file or media widget.
```
Comment (damiankloip) [inline file_dropzone.info]: nit: just file, media widget is now in the sub module.

File: file_dropzone.module (hunk 1 above)
Comment (fabianx-ai) [inline file_dropzone.module:411]: I think this should give a normal error for number of files exceeded, but I will check that. Good point.

File: file_dropzone.info (hunk 2 above)
Comment (fabianx-ai) [inline file_dropzone.info]: Good point.

**Pull request closed without merging**